src/SeqViewerContainer.tsx

```diff
@@ -91,6 +93,24 @@ class SeqViewerContainer extends React.Component<SeqViewerContainerProps, SeqVie
     };
   }
 
+  selectionIsProgramatic(selection: any): selection is Selection {
+    // If the selection was done programatically, it has not type
+    return !selection.type;
+  }
+
+  // If the selection prop updates, also scroll the linear view to the new selection
+  componentDidUpdate = (prevProps: SeqViewerContainerProps) => {
+    // Only scroll if the selection was done programatically
```

Comment:
```suggestion
    // Only scroll if the selection was done passed in as a prop by a user of SeqViz. Otherwise the selection was
    // made by the user clicking an element or selecting a range of sequences
```